business_rules/operators.py

```diff
@@ -923,6 +922,18 @@ def has_different_values(self, other_value: dict):
     def has_same_values(self, other_value: dict):
         return ~self.has_different_values(other_value)
 
+    @type_operator(FIELD_DATAFRAME)
+    def is_ordered_by(self, other_value: dict) -> pd.Series:
```

Comment:
Do we need to have a flag for descending vs. ascending?

Reply:
Added in the latest commit